trunk/README.txt

```diff
@@ -158,6 +158,10 @@ Filter the regular expression used to find URLs to be converted to https://go.pa
 
 == Changelog ==
 
```

Comment:
There's also a ver number at == Upgrade Notice == section.

Reply:
Thanks for pointing that out!